src/Umbraco.Web.BackOffice/Trees/ContentTreeController.cs

```diff
@@ -369,8 +369,11 @@ private void AddActionNode<TAction>(IUmbracoEntity item, MenuItemCollection menu
         }
 
         public async Task<EntitySearchResults> SearchAsync(string query, int pageSize, long pageIndex, string? searchFrom = null)
+            => await SearchAsync(query, pageSize, pageIndex, searchFrom);
+
+        public async Task<EntitySearchResults> SearchAsync(string query, int pageSize, long pageIndex, string? searchFrom = null, string? culture = null)
         {
-            var results = _treeSearcher.ExamineSearch(query, UmbracoEntityTypes.Document, pageSize, pageIndex, out long totalFound, searchFrom);
+            var results = _treeSearcher.ExamineSearch(query, UmbracoEntityTypes.Document, pageSize, pageIndex, out long totalFound, culture: culture, searchFrom: searchFrom);
```

Comment:
Hmmm this is interesting, seems like we call the same method recursively on line 372, but we're really calling the new action with an optional parameter, wonder how the compiler handles this 🤔 This would also be worth at test for packages that has compiled against a previous version i think 🕵️